packages/devtools_app/lib/src/ui/search.dart

```diff
@@ -175,17 +172,17 @@ mixin SearchControllerMixin<T extends DataSearchStateMixin> {
       matchIndex.value = 1; // first item because [matchIndex] us 1-based
     }
     _activeSearchMatch.value?.isActiveSearchMatch = false;
-    _activeSearchMatch.value = searchMatches.value[activeMatchIndex]
+    _activeSearchMatch.value = searchMatches.value[activeMatchIndex]!
       ..isActiveSearchMatch = true;
     onMatchChanged(activeMatchIndex);
   }
 
   /// Delay to reduce the amount of search queries
   /// Duration.zero (default) disables debounce
-  Duration get debounceDelay => null;
+  Duration? get debounceDelay => null;
 
   List<T> matchesForSearch(
-    String search, {
+    String? search, {
```

Comment:
make this non-nullable. If there are places where a null value is possibly being passed in, default to the empty string `?? ''`

Reply:
done